rdrf/rdrf/exporter.py

```diff
@@ -499,6 +505,13 @@ def _get_context_form_groups(self):
             data.append(cfg_dict)
         return data
 
+def cull_defaults(Model, d):
+    for field in Model._meta.fields:
+        if (field.name in d and
+            (d[field.name] == field.default or
+             (not d[field.name] and field.default == fields.NOT_PROVIDED))):
```

Comment:
It was because CharFields have an implicit default of `""`.

I think I should replace `not d[field.name]` with `d[field.name] in (None, "")`.

Anyway, this is the least urgent PR out of the list. Main point of this was to illustrate what the yaml looks like without redundant noise.